React/exercises/identifying-components.md

```diff
@@ -0,0 +1,13 @@
+# Identifying Components
+In this exercise, we will go through a popular website and brainstorm how to break down the web application into components.
+
+To get you started, we've done an example of this using the MediaRanker application that you're already familiar with.
+
+![Media Ranker as Components](../images/media-ranker-components.png)
```

Comment:
This is pretty minor, but the image suggests that the Navigation component contains the MediaSplash component (and thus all other parts of the page except the logo). This doesn't seem like a normal React component setup.